tools/sgen/input/StylesheetsConfig.yml

```diff
@@ -0,0 +1,20 @@
+objc: true
+generateBaseStyle: true
+inputDir: ./
+inputs:
+  - FluentUIStyle.yml
```

Comment:
Can we pass more than one style to sgen? If so, what would be a use case to break it down? Do we need to modularize it?

Reply:
It is possible to pass more than one style to sGen in order to create different/multiple themes (which won't be the FluentUI case). But it's supported for consumers in case they want to have more than one theme.